packages/typescript-private/host-agent/src/processor/command.process-registry.ts

```diff
@@ -53,6 +53,12 @@ export class CommandProcessRegistry {
         };
         child.on('spawn', () => {
           this.logger.info('child process spawned', { command: commandReplaced, args: argsReplaced });
+          const stepEventhandler = eventHandler as StepMessageEventHandler;
```

Comment:
please consider of change "as" to "instanceof" because of "as" does not check instance's type